### PR TITLE
Feat at all

### PR DESCRIPTION
**Pull request by hcfw007**
https://github.com/wechaty/wechaty/issues/2394

I can continue working on wechaty pr and test afther this one accepted.

### CODE REVIEW

**Comment by hcfw007** [comment]
@huan ping

**Comment by hcfw007** [comment]
@huan ping

File: src/mixins/room-mixin.ts [hunk 1]
```diff
@@ -61,6 +61,8 @@ const roomMixin = <MixinBase extends typeof PuppetSkeleton & ContactMixin & Room
       */
     abstract roomAnnounce (roomId: string)               : Promise<string>
     abstract roomAnnounce (roomId: string, text: string) : Promise<void>
+    abstract roomAnnounce (roomId: string, text: string, isAnnounce: boolean) : Promise<string | void>
+    abstract roomAnnounce (roomId: string, text: string, isAnnounce: false, mentionList: string[]) : Promise<string>
```
Comment (huan) [inline src/mixins/room-mixin.ts:65]: I think the `isAnnounce` is unclear for what its meaning is. I remember you mentioned the `pin` I think it's a better name, how do you think about let's change this name from `isAnnounce` to `pin`?

And I forgot why we need to add `mentionList` here, can you explain it briefly? 

Thanks!

**Review by huan — CHANGES_REQUESTED** [review]

File: src/mixins/room-mixin.ts (hunk 1 above)
Comment (hcfw007) [reply src/mixins/room-mixin.ts:65]: This PR is outdated and should be closed.
As we talked before, we should use '@${RoomId}' to send @all messages.

**Pull request closed without merging**